FFXIV_TexTools_CLI/Configuration.cs

```diff
@@ -21,17 +21,17 @@ public void ReadConfig()
             string configFile = Path.Combine(_projectconfDirectory.FullName, "config.cfg");
             var configFileFromPath = new ConfigParser(configFile);
             var configFileFromString = new ConfigParser(@"[Directories]
-# Full path to game install, including 'Final Fantasy XIV - A Realm Reborn'
+# All paths can be written with or without escaping
+
+# Full path to game install, including 'FINAL FANTASY XIV - A Realm Reborn'
 GameDirectory
 
-# Full path to directory with your index backups
+# Full path to directory with your index backups, this can be any directory where you wish to store your backups
 BackupDirectory
 
 # Full path to directory where FFXIV.cfg and character data is saved, including 'FINAL FANTASY XIV - A Realm Reborn'
-ConfigDirectory
-
-# Full path to directory where your modpacks are located
-ModpackDirectory",
```

Comment:
This change isn't documented in the commits, to what I can see?

Reply:
This change accidentally went undocumented, as I was under the impression I had already pushed these changes before committing the documented changes. The config file has been updated to be more informative, and unused variables have been removed until they are implemented in the rest of the code.